tests/Jobs/InstanceSegmentationRequestTest.php

```diff
@@ -82,7 +82,10 @@ public function testHandleTrainingProposalWithLabelID()
             'available_bytes' => intval(8E+9),
             'max_workers' => 2,
             'tmp_dir' => $tmpDir,
-            'training_proposals' => [$image->id => [[11, 20, 30, $trainingProposal->label_id]], $image2->id =>[[14, 22, 33, $trainingProposal2->label_id], [15, 30, 20, $trainingProposal3->label_id]]],
+            'training_proposals' => [ $image->id => [[11, 20, 30, $trainingProposal->label_id]],
+                                      $image2->id =>[[14, 22, 33, $trainingProposal2->label_id],
+                                      [15, 30, 20, $trainingProposal3->label_id]]
+                                    ],
```

Comment:
I would prefer the following formatting for better readability:

```suggestion
            'training_proposals' => [
               $image->id => [
                  [11, 20, 30, $trainingProposal->label_id]
               ],
               $image2->id => [
                  [14, 22, 33, $trainingProposal2->label_id],
                  [15, 30, 20, $trainingProposal3->label_id],
               ],
```